lib/constants.go

```diff
@@ -949,6 +984,17 @@ var DeSoMainnetParams = DeSoParams{
 	// reserve ratios.
 	CreatorCoinAutoSellThresholdNanos: uint64(10),
 
+	// Unstaked stake can be unlocked after a minimum of N elapsed epochs.
+	StakeLockupEpochDuration: uint64(3),
+
+	// Jailed validators can be unjailed after a minimum of N elapsed epochs.
+	ValidatorJailEpochDuration: uint64(3),
+
+	// When registering, a validator must include a VotingSignatureBlockHeight within CurrentBlockHeight
+	// to CurrentBlockHeight + ValidatorVotingSignatureBlockHeightWindow blocks. This is to prevent
+	// validator registration replay attacks.
+	ValidatorVotingSignatureBlockHeightWindow: uint64(300),
```

Comment:
Wonder if we should make this value larger. When we shift to PoS we'll be doing 1 block per second so feels like 3600 might be better. On the fence, though, because 300s is also not bad.

Reply:
I think leave it as-is for now.